### PR TITLE
remove unused sidebar component

### PR DESCRIPTION
**Pull request by brad-decker**
The Sidebar component and app state are no longer used, the only place it was previously used was as part of the old cancel/speedup flow. This has since been replaced by the EIP-1559 experience.

### CODE REVIEW

**Review by NiranjanaBinoy — DISMISSED** [review]
LGTM

File: ui/store/actions.js [hunk 1]
```diff
@@ -1636,6 +1636,7 @@ export function showModal(payload) {
     payload,
   };
 }
+// comment
```
Comment (Gudahtt) [inline ui/store/actions.js:1639]: I can confirm that this is a comment

**Review by adonesky1 — DISMISSED** [review]
LGTM! Just looks like @Gudahtt has nit

File: ui/store/actions.js (hunk 1 above)
Comment (brad-decker) [reply ui/store/actions.js:1639]: hahahaha

**Review by darkwing — DISMISSED** [review]

File: ui/store/actions.js (hunk 1 above)
Comment (brad-decker) [reply ui/store/actions.js]: removed in latest.

**Review by darkwing — APPROVED** [review]

**Review by adonesky1 — APPROVED** [review]

**Pull request merged**